api/taskcluster/webhook.go

```diff
@@ -326,6 +337,9 @@ func createRun(client *http.Client, sha, api string, username string, password s
 	if labels != nil {
 		payload.Add("labels", strings.Join(labels, ","))
 	}
+	// Ensure we call back to this appengine version instance.
+	host := appengine.DefaultVersionHostname(ctx)
```

Comment:
I don't think this does what your comment says: it's the *default* version, not this (the current) version.

Reply:
What about `shared.GetHostname`?

Reply:
That's what I had intended to call, done.